Add getPlaidAccounts method

itsjohnward opened the pull request:
Copied the properties and their types straight from the documentation https://lunchmoney.dev/#plaid-accounts and mimicked the style of `getAssets()` and `getTransactions()` for consistency.

Also ran `npm run-script build` to update the dist/ directory. **Note**: that also pulled in the query string diff from https://github.com/lunch-money/lunch-money-js/pull/1

lunchbag commented:
@itsjohnward 
Sorry for the supremely late reply here!
We added a missing `display_name` property being returned, could you add that then we can merge this. Thanks!

itsjohnward commented:
@lunchbag, I just added `display_name`. I also noticed a few other missing `Asset` properties while I was at it: `balance_as_of`, `currency`, `closed_on`, and `created_at`. Added those as well.

One other thing is I'm seeing nulls getting output for optional fields like `subtype_name`, so I added `| null` for those. Otherwise typescript complains:

```
Type 'null' is not assignable to type 'string | undefined'.ts(2322)
index.ts(16, 2): The expected type comes from property 'subtype_name' which is declared here on type 'Asset'
```

lunchbag commented:
Thanks!! Will merge this now.

I'm curious to learn why you wanted this added here!
